.github/scripts/Configure-AppxManifest.ps1

```diff
@@ -53,18 +91,29 @@ elseif ($Branch -eq "Store")
     $xmlDoc.Package.Applications.Application.VisualElements.DefaultTile.ShortName="Files"
 
     # Remove an capability that is used for the sideload
-    $nsmgr = New-Object System.Xml.XmlNamespaceManager($xmlDoc.NameTable)
-    $nsmgr.AddNamespace("pkg", "http://schemas.microsoft.com/appx/manifest/foundation/windows10")
-    $nsmgr.AddNamespace("rescap", "http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities")
     $pm = $xmlDoc.SelectSingleNode("/pkg:Package/pkg:Capabilities/rescap:Capability[@Name='packageManagement']", $nsmgr)
     $xmlDoc.Package.Capabilities.RemoveChild($pm)
+
+    # Update app protocol and execution alias
+    $ap = $xmlDoc.SelectSingleNode("/pkg:Package/pkg:Applications/pkg:Application/pkg:Extensions/uap:Extension[@Category='windows.protocol']/uap:Protocol", $nsmgr)
+    $aea = $xmlDoc.SelectSingleNode("/pkg:Package/pkg:Applications/pkg:Application/pkg:Extensions/uap5:Extension[@Category='windows.appExecutionAlias']/uap5:AppExecutionAlias/uap5:ExecutionAlias", $nsmgr)
+    $ap.SetAttribute("Name", "files");
+    $aea.SetAttribute("Alias", "files.exe");
```

Comment:
Considering we've decided to keep `files.exe` for backwards compatibility, will this cause issues?

Reply:
Maybe, maybe not. If that might happen we could remove that in Stable/Store release,

Reply:
I think we need to remove it for the stable release

Reply:
Done. Btw I renamed it to files-dev, files-preview and files instead of files-uwp. This alone doesn't need backward compatibility?

Reply:
Unless @hishitetsu sees an issue with it, I think `files-uwp` can be removed completely.

Reply:
I think `files-uwp` should also be kept for backward compatibility. It is not a public feature, but someone may have written scripts using this URI scheme.
Also, I think `files` is a too generic term for a URI scheme and may be confused with `file` URI scheme. So I think `files-app`, `files-app-stable`, `files-app-preview`, `files-app-dev` are better.

Reply:
I personally prefer `files`, as it's short and easy to type, but I'm not set on it.

Reply:
Easy to change before we merge this.
I'll test this again later.

Reply:
> Also, I'm not sure we can set more than one protocol.

I've confirmed that it works fine even if the app has mutiple protocols.

I believe each build should have these aliases and URI schemes.
* Alias

|Build|files.exe|files-stable.exe|files-preview.exe|files-dev.exe|
|---|---|---|---|---|
|Stable|o|o| | |
|Preview|o| |o| |
|Dev|o| | |o|

* URI scheme

|Build|files:|files-uwp:|files-stable:|files-preview:|files-dev:|
|---|---|---|---|---|---|
|Stable|o|o|o| | |
|Preview|o|o| |o| |
|Dev|o|o| | |o|